classes/Connector/MySQLDBConnector.php

```diff
@@ -13,20 +13,16 @@ public function __construct($connectionConfig = null)
         parent::__construct("MySQL");
         require_once($this->getAdaptersDirectory() . '/OptionAdapter.php');
     }
-
-    public function test_wp_die_callback() {
-        return array( $this, 'test_die_handler' );
-    }
-
-    public function test_die_handler($message, $title = '', $args = array()) {
-        throw new Exception("DB Connection failed");
-    }
     
     public function TestConnection()
     {
         error_reporting(E_ALL ^ E_WARNING);
-        add_filter('wp_die_handler', array($this, 'test_wp_die_callback'));
-        $connection = $this->createConnection();
+        $connectionConfig = $this->connectionConfig;
+        $password = $this->decryptString($connectionConfig['password']);
+        $newWpdb = new wpdbCustom($connectionConfig['user'], $password, $connectionConfig['name'], $connectionConfig['hostname']);
+        if (!$newWpdb->has_connected) { // Database Error
+            throw new Exception("DB Connection failed");
```

Comment:
@carmelopreeostudios please update the message as quoted in the task "Connection failed. Please check your connection details"